components/contextualhelp/index.css

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-ContextualHelp {
+  /* Layout Variables */
+  --spectrum-contextual-help-minimum-width: var(--spectrum-contextual-help-minimum-width);
+  --spectrum-contextual-help-padding: var(--spectrum-spacing-400);
+  --spectrum-contextual-help-content-spacing: var(--spectrum-spacing-100);
+  --spectrum-contextual-help-link-spacing: var(--spectrum-spacing-300);
+
+  /* Typography Variables */
+  /* TODO: these need to be changed to the contextual-help tokens when available */
+  --spectrum-contextual-help-title-size: var(--spectrum-heading-size-xs);
+  --spectrum-contextual-help-body-size: var(--spectrum-body-size-s);
+
+
+
+  /* Mobile styling */
+  @media screen and (max-width: 700px) {
+    --spectrum-contextual-help-content-spacing: var(--spectrum-spacing-200);
+  }
+
+}
+
+.spectrum-ContextualHelp {
+  position: relative;
+  min-inline-size: var(--mod-spectrum-contextual-help-minimum-width, var(--spectrum-contextual-help-minimum-width));
+}
+
+.spectrum-ContextualHelp-button {
+  display: block;
+}
+
+.spectrum-ContextualHelp-popover {
+  padding-block: var(--mod-spectrum-contextual-help-padding, var(--spectrum-contextual-help-padding));
+  padding-inline: var(--mod-spectrum-contextual-help-padding, var(--spectrum-contextual-help-padding));
+  font-size: var(--mod-spectrum-contextual-help-body-size, var(--spectrum-contextual-help-body-size));
+
+
+  h2, p {
```

Comment:
Can we be sure that an h2 will always be the appropriate heading level for this component? What if it doesn't make sense in the hierarchy of the page? Do we need to assign these styles to a class instead of a specific h-tag? What if the product needs to use a span or other typographical element for SEO? Some things to ponder.

Reply:
Good call. I'll update this to a class